digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <div v-if="!isLoaded" class="main-menu">
+      <h1>Digital Deck</h1>
+      <br />
+      <span>Decks</span><br />
+      <input v-model.number="deckSelected" type="number" /><br />
+      <span>Cards on hand</span><br />
+      <input v-model.number="cardsPerPlayer" type="number" /><br />
+      <span>Cards on table</span><br />
+      <input v-model.number="cardsOnTable" type="number" /><br /><br />
+      <button @click="submitForm" style="cursor: pointer;">Start</button>
+      <p v-for="error of v$.$errors" :key="error.$uid">
+        <strong>{{ error.$message }}</strong>
+        <span v-if="error.$property === 'cardsPerPlayer'"
+          ><strong> for cards on hand</strong></span
+        >
+        <span v-else-if="error.$property === 'cardsOnTable'"
+          ><strong> for cards on table</strong></span
+        >
+        <span v-else><strong> for number of decks</strong></span>
+      </p>
+    </div>
+    <game v-if="isLoaded" />
```

Comment:
I'm not sure how to feel about having the `Game` component be inside `MainMenu`. Since we're running out of time, it can stay for now, but I don't really like it.

Reply:
Would be happy to hear of better way to do this. We can discuss this.